docs/howto/krustlet-on-kubernetes-with-inlets.md

```diff
@@ -0,0 +1,265 @@
+# Running Krustlet on Kubernetes with inlets
+
+These are steps for running a krustlet node on your own computer. You can run any kind of 
+Kubernetes cluster you like, whether that's local on your computer or remote in a managed 
+Kubernetes offering.
+
+## Prerequisites
+
+There are a number of ways to give the Kubernetes API server access to the krustlet's API. Various methods include using a VPN, a VM within the Kubernetes cluster's private network, or a tunnel. Inlets is a popular open source service tunnel and proxy that is listed on the CNCF Landscape. It can be used to forward the port of the krustlet to the Kubernetes cluster so that the API server can access it as if it were deployed within the cluster directly.
+
+The tunnel has two components. A client which runs on your local machine, and a server which is deployed as a Pod inside the Kubernetes cluster. The client connects to the server and provides a persistent link.
+
+* [inlets - "The Cloud Native Tunnel"](https://docs.inlets.dev/)
+
+Run without `sudo` to download the binary to your local directory, then move it to your `PATH`.
```

Comment:
even without `sudo`, this still poses a massive security risk. `rm -rf $HOME` on random shell scripts is a thing.

Have a look at https://chocolatey.org/install for a great example how other projects recommend running untrusted shell scripts on the internet.

Reply:
What are you asking me to do? How do we unblock the PR?

Reply:
It's getting very late here (almost 11pm) and I'm trying to get the PR "over the line", if possible.

I've removed the installation instructions and pointed at the releases page.

Reply:
I would just provide a link to the install guide on inlet's site. Leave the responsibility to the inlets project on how they recommend how to install their project.

Reply:
https://github.com/inlets/inlets#get-started

Reply:
If we need to revisit in the morning, lets.

Reply:
I've done the link to the releases page.